### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kevensen**
Adding two tests to helm

- forseti inventory list
- forseti model list

These can be invoked by `helm test forseti`

Please do not merge before #19

### CODE REVIEW

File: charts/forseti-security/templates/tests/inventory-list-test.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    command: ['forseti']
+    args:
+    - --endpoint
+    - {{ .Release.Name }}-server.{{ .Release.Namespace }}.svc.cluster.local:50051
+    - inventory
+    - list
+    env:
+    - name: FORSETI_HOME
+      value: /forseti-security/
+  restartPolicy: Never
```
Comment (hshin-g) [inline charts/forseti-security/templates/tests/inventory-list-test.yaml]: Should there be a newline here?

File: charts/forseti-security/templates/tests/model-list-test.yaml [hunk 2]
```diff
@@ -12,28 +12,27 @@
...
+    - list
     env:
     - name: FORSETI_HOME
       value: /forseti-security/
-    securityContext:
-      runAsUser: 1000
-  restartPolicy: Never
-  schedulerName: default-scheduler
\ No newline at end of file
+  restartPolicy: Never
```
Comment (hshin-g) [inline charts/forseti-security/templates/tests/model-list-test.yaml]: Should there be a newline here?

File: charts/forseti-security/templates/tests/inventory-list-test.yaml (hunk 1 above)
Comment (kevensen) [reply charts/forseti-security/templates/tests/inventory-list-test.yaml]: Yup.  Will fix

File: charts/forseti-security/templates/tests/model-list-test.yaml (hunk 2 above)
Comment (kevensen) [reply charts/forseti-security/templates/tests/model-list-test.yaml]: Yup.  Will fix.

**Pull request closed without merging**